src/BinaryCoding.h

```diff
@@ -175,4 +176,11 @@ inline uint64_t decode64BE(const uint8_t* _Nonnull src) {
     // clang-format on
 }
 
+/// Encodes an ASCII string prefixed by the length (varInt)
+inline void encodeString(const std::string& str, std::vector<uint8_t>& data) {
```

Comment:
Is this common enough to stay in `src/BinaryCoding`?

Reply:
I think so.  It is quite generic.